src/cljs/main/broadfcui/page/method_repo/methods_configs_acl.cljs

```diff
@@ -111,8 +113,11 @@
               :payload non-empty-acls-w-public
               :on-done (net/handle-ajax-response
                          (fn [{:keys [success? parsed-response] :as response}]
-                           (swap! state assoc :acl-response response)
-                           (swap! state dissoc :saving?)))})))))
+                           (if success?
+                             (modal/pop-modal)
+                             (do
+                               (swap! state assoc :persist-error (:message parsed-response))
+                               (swap! state dissoc :saving?)))))})))))
```

Comment:
Each state update triggers a re-render. It would be better to either assoc `:saving? false` along with the other assoc, or if you want to clean up the key, update in one go:

```clojure
(swap! state #(-> %
                  (assoc :persist-error (:message parsed-response))
                  (dissoc :saving?)))
```